framework/caching/Cache.php

```diff
@@ -462,7 +474,9 @@ protected function getValues($keys)
     {
         $results = [];
         foreach ($keys as $key) {
+            $this->beginProfile($key, __METHOD__);
```

Comment:
This is part of public `multiGet`. If I move what will be key for `beginProfile`?